lib/aasm/instance_base.rb

```diff
@@ -14,15 +14,13 @@ def current_state
 
     def current_state=(state)
       @instance.aasm_write_state_without_persistence(state, @name)
-      # @current_state = state
     end
 
     def enter_initial_state
       state_name = determine_state_name(@instance.class.aasm(@name).initial_state)
       state_object = state_object_for_name(state_name)
 
       state_object.fire_callbacks(:before_enter, @instance)
-      # state_object.fire_callbacks(:enter, @instance)
```

Comment:
@anilmaurya When I added this TODO, I wasn't sure if we still need/want to support this callback. Reason is that actually there is no difference between the `enter` and the `before_enter` callback (there is nothing happening in between). It's just a semantic difference (redundant).

What do you think?

As this would be a DSL change, I initially wanted to make that change for _AASM 4_.